docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_s3.rst

```diff
@@ -3,16 +3,85 @@
 How to configure a Validation Result store in S3
 ================================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will help you configure a new storage location for Validations on Amazon S3.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectation Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Installed `boto3 <https://github.com/boto/boto3>`_ in the environment where you will run Great Expectations.
+    - Identified the S3 bucket and prefix where Validation results will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
+1. **Identify your Data Context Validations Store**
```

Comment:
I think we need a step to configure credentials for boto, even if that step is primarily just a link to boto's documentation..